### PR TITLE
[Merged by Bors] - Fix/node selector

### PR DESCRIPTION
**Pull request by Maleware**
# Description

Add role selector to airflow operator

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,13 @@
...
 
+- Fixed the RoleGroup `selector`. It was not used before. ([#208])
+- `operator-rs` `0.27.1` -> `0.30.1` ([#208])
 - Updated stackable image versions ([#193]).
 - `operator-rs` `0.25.2` -> `0.27.1` ([#197]).
 - [BREAKING] Use Product image selection instead of version. `spec.version` has been replaced by `spec.image` ([#206]).
 
+[#208]: https://github.com/stackabletech/airflow-operator/pull/208
 [#197]: https://github.com/stackabletech/airflow-operator/pull/197
 [#206]: https://github.com/stackabletech/airflow-operator/pull/206
```
Comment (maltesander) [inline CHANGELOG.md:15]:
```suggestion
[#197]: https://github.com/stackabletech/airflow-operator/pull/197
[#206]: https://github.com/stackabletech/airflow-operator/pull/206
[#208]: https://github.com/stackabletech/airflow-operator/pull/208
```

**Review by maltesander — APPROVED** [review]
LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (Maleware) [reply CHANGELOG.md]: Done, thanks!

**Comment by Maleware** [comment]
DO NOT MERGE. TESTS FAILED

**Comment by Maleware** [comment]
bors merge

**Pull request closed without merging**